vscode/src/circuit.ts

```diff
@@ -17,137 +19,307 @@ import { loadProject } from "./projectSystem";
 import { EventType, UserFlowStatus, sendTelemetryEvent } from "./telemetry";
 import { getRandomGuid } from "./utils";
 import { sendMessageToPanel } from "./webviewPanel";
+import { ProgramConfig } from "../../npm/qsharp/dist/compiler/compiler";
```

Comment:
We shouldn't be reaching into the npm code directly via paths like this. We should be importing from qsharp-lang (as per line 6).

Reply:
Auto-import strikes again! Fixed.